### PR TITLE
Fix validation pattern for portnum type

### PR DESCRIPTION
**Pull request by rhafer**
The pattern failed for all port that specified the tcp protocol.

Issue #1217

Fixes # .

Changes proposed in this pull request:
*
*

### CODE REVIEW

File: kiwi/xml_parse.py [hunk 1]
```diff
@@ -5074,7 +5074,7 @@ def validate_portnum_type(self, value):
             if not self.gds_validate_simple_patterns(
                     self.validate_portnum_type_patterns_, value):
                 warnings_.warn('Value "%s" does not match xsd pattern restrictions: %s' % (value.encode('utf-8'), self.validate_portnum_type_patterns_, ))
-    validate_portnum_type_patterns_ = [['^(\\d+$|^\\d+/(udp$|^tcp))$']]
+    validate_portnum_type_patterns_ = [['^(\\d+$|\\d+/(udp|tcp))$']]
```
Comment (aplanas) [inline kiwi/xml_parse.py]: is not this file autogenerated?

Comment (davidcassany) [reply kiwi/xml_parse.py]: yes the file is autogenerated. The regular expression is taken from https://github.com/OSInside/kiwi/blob/368f44262cb56c448b101a373003efca4a790a02/kiwi/schema/kiwi.rnc#L37

kiwi.rng is autogenerated too :)

Comment (schaefi) [reply kiwi/xml_parse.py]: Yes the Relax schema is maintained in its condensed format (RNC) Thus schema changes must be performed on kiwi.rnc. Once a change there is made call:

```
make kiwi/schema/kiwi.rng
```

This builds the RNG schema as well as auto generates the data structures. The make call must be performed from within a tox development environment. For details read:

* http://osinside.github.io/kiwi/development.html#create-a-python-virtual-development-environment

Thanks

**Comment by davidcassany** [comment]
Ok I have just validated that by using the latest generateDS the regular expression is translated as expected.

@schaefi I believe it might be time to test a newer version of generateDS and see if it works fine. This not obvious to test, but I'd say that some local tests and the integration tests should be sufficient (note that we should check no warnings are shown within the integration tests logs).

What do you think?

File: kiwi/xml_parse.py (hunk 1 above)
Comment (rhafer) [reply kiwi/xml_parse.py]: > is not this file autogenerated?

Ah, ok. Didn't know that.

**Pull request closed without merging**